docs/refs.bib

```diff
@@ -271,6 +271,16 @@ @misc{Cariello_2013_Weak_irreducible
   primaryClass={quant-ph}
 }
 
+
+@article{campos2024,
```

Comment:
Could you move this to L134 after `Cabello_2002_NParticle`? And keep the naming style the same as the other references?